Backend/starter_project_mavericks/BlogApp/BlogApp.Application/Features/Ratings/CQRS/Commands/Update_RatingCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BlogApp.Application.Features.Ratings.DTOs;
+using BlogApp.Application.Responses;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Features.Ratings.CQRS.Commands;
+public class Update_RatingCommand : IRequest<BaseResponse<int>>
+{
+    public int BlogId { get; set; }
+    public RatingDto RatingDto { get; set; }
```

Comment:
The same comment I have added to create rating applies here too. Create an UpdateRateDto and add your BlogId and RaterId there and use that to create your UpdateRatingCommand.

And one more thing here, you should remove the "_" in your file name. it shuould be "UpdateRatingCommand.cs"